### DISCUSSION TITLE
[v3] $this->redirect and laravel redirect() always use navigate to jump

### DISCUSSION THREAD

**Discussion by killjin** [livewire/livewire#5803] [7. Bugs (archived)] [answered]
![image](https://github.com/livewire/livewire/assets/2804854/e5b8cf08-3c58-497d-ae11-b19685ce1484)

see livewire.js,  alpine:navigated  set isNavigating = true;

PHP:
```php
$this->redirect($url);
```
This will force use Alpine.navigate? 

1. This will use the html replace instead of redirecting to the url.
2. This will cause the URl to not change, causing some weird display issues, @Livewire Lazy components(only Lazy compoments) won't load on new page(livewire/update does not execute).

![image](https://github.com/livewire/livewire/assets/2804854/005e2242-2ccb-485e-925c-c2b046552acd)
![image](https://github.com/livewire/livewire/assets/2804854/fbbe4e24-e68d-4aec-93cc-03280af24112)

**Comment by schonhoff** [comment]
Would love to see an option to deactivate it on redirecting even if the route to redirect to is an Livewire component. Maybe another parameter would be possible?.

**Comment by PhiloNL** [comment]
I think this is a duplicate of https://github.com/livewire/livewire/discussions/5786 and is probably a bug.
A failing test is added: https://github.com/livewire/livewire/pull/5787

**Reply by schonhoff** [reply]
The discussion is for Livewires `$this->redirect()` method and not the Laravel `redirect` method. It uses the Alpine navigate, which is fine for many, but in my case it triggers the `<noscript>` tag every time. (maybe that should be a new discussion...) But it also filckers for me and sometimes people don't want the SPA-feeling for redirecting.

**Comment by calebporzio** [comment, marked as answer]
Sorry, this was a bug and has been fixed in beta-4

Livewire will only use navigate to redirect to a new page if it has already been used on that page

**Comment by onemoreahmad** [comment]
I think it's not 100% fixed. 

**`wire:navigate` and then redirecting to a subdomain doesn't work.** 

When I go to /login page (full page component) using wire:navigate
and then submit the form and redirect to `subdomain.dashboard.com`, it dose not work, I got 404 error. ❌

but if I refresh the /login page before submitting the form and then submit it, it does work and redirect me to the subdomain. ✅

If I delete `wire:navigate` in /login link tag and go to login without `wire:navigate`, it works and redirect me to subdomain. ✅

**Reply by michael-rubel** [reply]
I can confirm that it still doesn't work with external links when redirecting for the first time.

**Reply by OdunlamiZO** [reply]
Any update on this? I am having similar issue.
